docs/src/main/paradox/client-side/configuration.md

```diff
@@ -0,0 +1,51 @@
+# Configuration
+
+HTTP client settings are split into different sections  
+ 
+ * `akka.http.client`: basic client settings
+ * `akka.http.host-connection-pool`: pool settings
+ 
+## Basic Client Settings
+
+These settings influence the basic library behavior for each HTTP connection. When changed in the application's
+`application.conf` they globally affect the default client behavior.
+
+Basic client settings can be overridden in multiple ways:
+
+ * by passing custom @apidoc[ClientConnectionSettings] instances to APIs in @scala[@apidoc[Http$]]@java[@apidoc[Http]]
```

Comment:
The `$` is automatically chopped off for the java apidoc, so this should be equivalent:
```suggestion
 * by passing custom @apidoc[ClientConnectionSettings] instances to APIs in @apidoc[Http$]
```

Reply:
Ah, cool, copied that from somewhere else...